funnel/templates/project_cfp.html.jinja2

```diff
@@ -1,7 +1,8 @@
 {% extends "formlayout.html.jinja2" %}
 {% from "baseframe/forms.html.jinja2" import renderfield, rendersubmit, ajaxform %}
 {% from "baseframe/components.html.jinja2" import faicon %}
-{% block title %}Add proposal guidelines and timing{% endblock %}
+{% block title %}{% trans %}Add proposal guidelines and timing{% endtrans %}{% endblock %}
+{% block description %}{{ project.title }}{% if project.tagline %} &ndash; {{ project.tagline|striptags }}{% endif %}{% endblock %}
```

Comment:
I'm not sure this description should be included in functional pages. It's appropriate for content pages like the project landing page.